tests/security/pam/pam_su.pm

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: PAM tests for su, su to root should fail if user is not in group "wheel"
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#70345, tc#1167579
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use base 'consoletest';
+use utils 'clear_console';
+
+sub run {
+    select_console 'root-console';
+
+    # User will not be able to su to root since it is not belong to group "wheel"
+    my $user   = 'bernhard';
+    my $passwd = 'nots3cr3t';
+    my $group  = 'wheel';
+    validate_script_output "id $user | grep $group || echo 'check pass'", sub { m/check pass/ };
+
+    # Modify the PAM configuration files
+    my $su_file      = '/etc/pam.d/su';
+    my $su_file_bak  = '/tmp/su';
+    my $sul_file     = '/etc/pam.d/su-l';
+    my $sul_file_bak = '/tmp/su-l';
+    assert_script_run "cp $su_file $su_file_bak";
+    assert_script_run "cp $sul_file $sul_file_bak";
+    assert_script_run "sed -i '\$a auth     required       pam_wheel.so use_uid' $su_file";
+    assert_script_run "sed -i '\$a auth     required       pam_wheel.so use_uid' $sul_file";
+
+    # Switch to user console
+    clear_console;
+    select_console 'user-console';
+
+    # Then su to root should fail
+    assert_script_run "expect -c 'spawn su - root;
+expect \"Password: \"; send \"$passwd\\r\";
+'";
+
```

Comment:
Did you check the openQA log to make sure
the `su - root` is failed on purpose but not `expect` code's mistakes?
I checked the log it seems no expect error （http://10.67.17.9/tests/277#step/pam_su/22）.

Reply:
I will double check this, and add the verify code

Reply:
> I will double check this, and add the verify code

done